jwst/fits_generator/create_dms_data.py

```diff
@@ -3,7 +3,7 @@
 import os
 import numpy
 from astropy.io import fits as pyfits
-import jwst.fits_generator
+from .. import fits_generator
 
```

Comment:
I think this might also need to be at the same level: "from . import fits_generator". It's a module in the same directory.